Extensions/ExtendedMath.json

```diff
@@ -763,11 +788,12 @@
       "objectGroups": []
     },
     {
-      "description": "Rounds a value to a specified number of decimal places.",
+      "description": "Formats a number to use the specified number of decimal places. (Deprecated).",
```

Comment:
There is a typo: ". ("?

Reply:
Perhaps, I'm not sure what the right way is. I will remove that period.